Making Makefile on Ubuntu gives error

sudo: dnf: command not found
Makefile:5: recipe for target 'install-dependencies' failed
make: *** [install-dependencies] Error 1

![Screenshot from 2020-03-03 04-27-09](https://user-images.githubusercontent.com/43514828/75725934-6b9fa400-5d07-11ea-9bb4-f3faaa5f4234.png)

--- a/Makefile
+++ b/Makefile
@@ -1,10 +1,14 @@
 CURDIR := ${CURDIR}
 IMAGE ?= usercont/packit-dashboard:stg
-
 install-dependencies:
-	sudo dnf -y install python3-flask npm
+	if [ -f "/etc/redhat-release" ];\
+	 then\
+		sudo dnf -y install python3-flask npm;\
+	elif [ -f "/etc/debian_version" ];\
+	 then\
+		sudo apt -y install python3-flask npm;\
+	fi
 	npm install
-
 run:
 	FLASK_ENV=development FLASK_APP=packit_dashboard.app flask-3 run
 

